omnigan/generator.py

```diff
@@ -319,10 +319,12 @@ def forward(self, z, cond=None):
 
         d = super().forward(z)
 
-        return F.interpolate(
+        preds = F.interpolate(
             d, size=self._target_size, mode="bilinear", align_corners=True
         )
 
+        return preds, None
```

Comment:
This is so G.decoders["d"] always return two things no matter what the decoder is.